### PR TITLE
Updates mesos.rb with latest mesos version

### PR DESCRIPTION
**Pull request by philosoralphter**
I don't know if this is sufficient to make the change--I've never made a homebrew package.

### CODE REVIEW

File: Library/Formula/mesos.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Mesos < Formula
   homepage "http://mesos.apache.org"
-  url "http://mirror.cogentco.com/pub/apache/mesos/0.20.1/mesos-0.20.1.tar.gz"
+  url "http://mirror.cogentco.com/pub/apache/mesos/0.21.0/mesos-0.21.0.tar.gz"
   sha1 "8028366a2538551daaf290f7c62c4c8bfb415f61"
```
Comment (DomT4) [inline Library/Formula/mesos.rb]: Need to update the SHA1 as well :).

**Comment by DomT4** [comment]
You should be able to remove lines 15-18, and line 27. Upstream merged a patch on that a few months back that should be in this release. Aside that, looks good! :smiley:

**Comment by philosoralphter** [comment]
Thanks @DomT4.  I'll do that now.  One question: the sha1.  Is that of the downloaded tar ball?  If so, what is the preferred way to get that?  Locally after download or with an online tool and the url?

**Comment by DomT4** [comment]
Yeah. The way I usually get the SHA1 if it isn't displayed on the website is:

```
cd ~/Downloads && curl -O http://mirror.cogentco.com/pub/apache/mesos/0.21.0/mesos-0.21.0.tar.gz
openssl sha1 mesos-0.21.0.tar.gz
```

**Pull request closed without merging**